topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.md

```diff
@@ -0,0 +1,185 @@
+  - Quantitative analysis of peptide sequence properties
+time_estimation: '20m'
+contributors:
+   - jaidevjoshi83
+   - blankenberg
+---
+
+# Introduction
+{:.no_toc}
+
+
+Several computational methods have been proven very useful in the initial screening and prediction of peptides for various biological properties. These methods have emerged as effective alternatives to the lengthy and expensive traditional experimental approaches. In this tutorial, we will be discussing how to analyze peptide libraries on the basis of quantitative properties. In this tutorial, we will learn how to use different utilities of PDAUG to calculate various peptide-based features and utilize these features for various informative plots.
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+## Peptide Data
+
+In this step, we will retrieve the inbuild dataset, which contains anti-microbial (AMPs) and transmembrane peptides (TMPs).
+
+
+> ### {% icon hands_on %} Hands-on: Fetching inbuild data
+> 
+> 1. {% tool [PDAUG Peptide Data Access](toolshed.g2.bx.psu.edu/repos/jay/pdaug_peptide_data_access/pdaug_peptide_data_access/0.1.0) %} with the following parameters:
+>    - *"Datasets"*: `AMPvsTM` 
+>
+{: .hands_on}
+
+
+## Converting tabular data into fasta format
+
+In this step, we will be converting and splitting the tabular data file into fasta format. This tool splits data into two files based on their class labels.
+
+> ### {% icon hands_on %} Hands-on: Converting tabular data into fasta formate
+> 
+> 1. {% tool [PDAUG TSVtoFASTA](toolshed.g2.bx.psu.edu/repos/jay/pdaug_tsvtofasta/pdaug_tsvtofasta/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `PDAUG Peptide Data Access - AMPvsTM (tabular)` (output of **PDAUG Peptide Data Access** {% icon tool %})
+>    - *"Data conversion"*: ` WithClassLabel `
+>
+>
+{: .hands_on}
+
+## Analyzing peptide libraries (AMPs and TMPs) based on features and feature space 
+
+### Summary Plot for peptide libraries 
+
+In this step, we utilize **PDAUG Peptide Sequence Analysis** tool to compare peptide sequences based on hydrophobicity, hydrophobic movement, charge, amino acid fraction, and sequence length and creates a summary plot.
+
+
+
+> ### {% icon hands_on %} Hands-on: Generating a summary plot to assess peptide dataset
+> 
+> 1. {% tool [PDAUG Peptide Sequence Analysis](toolshed.g2.bx.psu.edu/repos/jay/pdaug_peptide_sequence_analysis/pdaug_peptide_sequence_analysis/0.1.0) %} with the following parameters:
+>    - *"Analysis options"*: `Plot Summary`
+>        - {% icon param-file %} *"First input file"*: `PDAUG TSVtoFASTA on data 1 - first (fasta)` (first output of **PDAUG TSVtoFASTA** {% icon tool %})
+>        - {% icon param-file %} *"Second input file"*: `PDAUG TSVtoFASTA on data 1 - second (fasta)` (second output of **PDAUG TSVtoFASTA** {% icon tool %})
+>        - *"first input file"*: `AMP`
+>        - *"Second input file"*: `TM `
+>
```

Comment:
It would be nice to use either TM or TMP consistently in the training instead of switching between both abbreviations.

Reply:
Correted